package.json

```diff
@@ -69,7 +71,7 @@
     "stage-branch": "export DEPLOY_SUBDIR=\"panoptes-front-end/$(git symbolic-ref --short HEAD)\"; npm run stage;  echo \"Deployed to http://$(git symbolic-ref --short HEAD).pfe-preview.zooniverse.org/\";",
     "stage-branch-with-docker": "./bin/run-through-docker.sh 'npm run-script stage-branch'",
     "stage-with-docker": "./bin/run-through-docker.sh 'npm run-script stage'",
-    "start": "check-engines && ./bin/serve.sh",
```

Comment:
no, it just annoys me and i forget to re-add it before committing. i'll make sure that's not in what gets merged